ios/styles/LinkStyle.mm

```diff
@@ -239,7 +239,7 @@ - (NSRange)getFullLinkRangeAt:(NSUInteger)location {
     inRange:inputRange
   ];
   
-  return manualLinkRange.length == 0 ? automaticLinkRange : manualLinkRange;
+  return manualUrl == nil ? automaticLinkRange : manualLinkRange;
```

Comment:
For the sake of safety I'd also assign `automaticUrl` to the automatic link range search result and do something like;
```suggestion
  return
    manualLink == nullptr
      ? automaticLink == nullptr ? NSMakeRange(0,0) : automaticLinkRange
      : manualLinkRange;
```